### DISCUSSION TITLE
Prevent Excessive Blocked Connections Statistics Entrie

### DISCUSSION THREAD

**Discussion by Cyber5imon** [tnodir/fort#129] [Q&A] [answered]
I have certain programs that are blocked and attempt outbound connections often. This causes the blocked connections tab of the statistics window to fill up very quickly. How can I prevent a program from spamming this window with hundreds if not thousands of needless entries? I don't need to see every blocked connection once I know that the program is attempting outbound connections and I've intentionally blocked it.

**Comment by tnodir** [comment, marked as answer]
Turn off the program's "Collect blocked connections" flag:

![program-log-blocked](https://github.com/tnodir/fort/assets/303655/a226c6ee-4dde-474d-a048-a00dab7354eb)
